src/twisted/web/_flatten.py

```diff
@@ -432,7 +432,9 @@ def flushBuffer() -> None:
             roots = []
             for generator in stack:
                 roots.append(generator.gi_frame.f_locals["root"])
-            roots.append(frame.f_locals["root"])
+            # Python 3.13 doesn't have "root" in f_locals:
```

Comment:
This was reported here  https://github.com/twisted/twisted/issues/12194

and there is [this comment that it also happens in 3.12.4](https://github.com/twisted/twisted/issues/12194#issuecomment-2173104263)